src/components/Calculator.js

```diff
@@ -1,68 +1,86 @@
+        <p style={{ userSelect: 'none' }}>5</p>
       </div>
-      <div className={styles.calculator__button__grey}>
-        <p>6</p>
+      <div className={styles.calculator__button__grey} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>6</p>
       </div>
-      <div className={styles.calculator__button__orange}>
-        <p>-</p>
+      <div className={styles.calculator__button__orange} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>-</p>
       </div>
-      <div className={styles.calculator__button__grey}>
-        <p>1</p>
+      <div className={styles.calculator__button__grey} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>1</p>
       </div>
-      <div className={styles.calculator__button__grey}>
-        <p>2</p>
+      <div className={styles.calculator__button__grey} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>2</p>
       </div>
-      <div className={styles.calculator__button__grey}>
-        <p>3</p>
+      <div className={styles.calculator__button__grey} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>3</p>
       </div>
-      <div className={styles.calculator__button__orange}>
-        <p>+</p>
+      <div className={styles.calculator__button__orange} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>+</p>
       </div>
-      <div className={`${styles.calculator__button__grey} ${styles.calculator__button__zero}`}>
-        <p>0</p>
+      <div className={`${styles.calculator__button__grey} ${styles.calculator__button__zero}`} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>0</p>
       </div>
-      <div className={styles.calculator__button__grey}>
-        <p>.</p>
+      <div className={styles.calculator__button__grey} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>.</p>
       </div>
-      <div className={styles.calculator__button__orange}>
-        <p>=</p>
+      <div className={styles.calculator__button__orange} onClick={clickHandler} onKeyPress={(e) => { if (e.key === 'Enter') clickHandler(); }} role="button" tabIndex={0}>
+        <p style={{ userSelect: 'none' }}>=</p>
```

Comment:
- [ ] (Optional) You can put all the symbols and numbers in an array as strings, then map through the array and display every symbol/number dynamically. This way, your code will be dry and there won't be repetition. You can read this [article](http://www.hackingwithreact.com/read/1/13/rendering-an-array-of-data-with-map-and-jsx) as a guide. 👍

Reply:
Thanks for the suggestion! I did an algorithm like that and added the required conditionals for class-naming. 🎉